### ISSUE TITLE
SKPhotoBrowser(originImage:photos:animatedFromView:) replacement

### ISSUE THREAD

**Issue by jdanthinne** [suzuki-0000/SKPhotoBrowser#363]
SKPhotoBrowser(originImage:photos:animatedFromView:) being marked as deprecated, what is the best alternative?

**Comment by aslanmehmetsalih** [comment]
i think `SKPhotoBrowser(photos: photos, initialPageIndex: initialPageIndex)` is best alternative.

**Comment by jdanthinne** [comment]
Yes, but what if I want to keep the zoom animation from the original tapped view?

**Comment by zenghaojim33** [comment]
+ 1

**Comment by derekcoder** [comment]
+1

**Comment by ajself** [comment]
The README does not reflect that this convenience initializer is marked as deprecated either, nor is there any documentation as to why.

**Comment by UzumakiAlfredo** [comment]
+1

**Comment by alfredcc** [comment]
+1

**Comment by buithuyen** [comment]
+1.
I was wondering about it too.

**Comment by RainManGO** [comment]
+1

**Comment by starkite** [comment]
+1

**Comment by LittleLittleKang** [comment]
I think we should use "SKPhotoBrowserDelegate", as shown in the sample code.

**Comment by tommyming** [comment]
manually close this issue due to inactivity, if you think this is worth to fix/improve, please feel free to reopen it or submit PR.
